doc/rst/language/spec/methods.rst

```diff
@@ -330,6 +330,102 @@ entirely, the receiver will be passed with a default intent. The default
    Given a variable ``x = 2``, a call to ``x.doubleMe()`` will set ``x``
    to ``4``.
 
+.. index::
+   single: methods; implicit this.
+.. _Implicit_this_dot_in_methods:
+
+Implicit *this.* in Methods
+---------------------------
+
+Within a method, an identifier can implicitly refer to a field or another
+method. The compiler will implicitly add a ``this.`` in such cases.
+
+   *Example (implicitThis.chpl)*.
+
+   In the below example, within ``proc R.method()``, ``field``,
```

Comment:
The commas here are a touch ambiguous to me on a first read. At first I thought "R.method", "field", etc were all part of the same list. I think it would be clearer to say:

```
... within ``proc R.method()``, the identifiers ``field``, ...
```